src/tests/test_update.py

```diff
@@ -128,15 +125,12 @@ def test_gob_event_action(self, mock_event):
 
     def test_gob_event_invalid_action(self, mock_event):
         # setup initial event and data
-        event, data = fixtures.get_event_data_fixure()
+        dummy_event, data = fixtures.get_event_data_fixure()
 
         last_event_expected = 1
         for action_expected in ['FOO', 'BAR']:
             data['_last_event'] = last_event_expected
-            event['action'] = action_expected
-
-            # from dictionary to object with attributes
-            dummy_event = fixtures.dict_to_object(event)
+            dummy_event.action = action_expected
```

Comment:
data = { ‘_last_event’: last_event_expected } is sufficient and removes the necessity to return two values in the fixture method